articles/Why-julia-is-slowly-replacing-python-for-machine-learning-and-data-science/index.md

```diff
@@ -17,38 +17,40 @@ images:
 
 ### Python Language Advantages
-Unlike Julia, Python is a general-purpose language. Despite not being built purposely for data science, Python offers many advantages to machine-learning and data scientists. Python has existed for around 30 years in which it has established strong relationships with many third-party packages. This has attracted many users. One of the drawbacks associated with Python is speed. But Python is implementing some great improvements, especially to the Python interpreter. The new [PyPy v7.1 interpreter](https://doc.pypy.org/en/latest/release-v7.1.0.html) is fast and reliable. Advances in parallel and multi-core processing are intended to make Python easier to speed up.
+Unlike Julia, Python is a general-purpose language. Despite not being built purposely for data science, Python offers many advantages to machine-learning and data scientists. Machine learning scientists and data scientists use python in sentiment analysis and natural language processing (NLP). This is because python libraries offer a convenient way to write highly performing algorithms. 
+
+Python has existed for around 30 years in which it has established strong relationships with many third-party packages. This has attracted many users. One of the drawbacks associated with Python is speed. But Python is implementing some great improvements, especially to the Python interpreter. The new [PyPy v7.1 interpreter](https://doc.pypy.org/en/latest/release-v7.1.0.html) is fast and reliable. Advances in parallel and multi-core processing are intended to make Python easier to speed up.
 
 ### Tailored for Machine Learning
-Python is used for a broad range of tasks. Julia, on the other hand, is primarily developed to perform machine learning and statistics tasks. Because Julia is explicitly made for high-level statistical work, it has several benefits over Python. In linear algebra, for example, "vanilla" Julia shows better performance than ["vanilla" Python](https://pypi.org/project/vanilla/). This is mainly because, unlike Julia, Python does not support all equations and matrices performed in machine-learning. While Python is a great language, especially with [NumPy](https://numpy.org/), Julia beats it when it comes to non-package experience with Julia being more catered towards machine learning calculations. The [operand system](https://docs.julialang.org/en/v1/manual/types/#Operations-on-Types) of Julia can only be compared with that of R. That of Python is a bit weaker regarding performance, and that is a big setback.
```

Comment:
I should've been more clear. I was referring to your link to the vanilla library for Python. It's not clear how that's related.